api/src/main/java/io/strimzi/api/kafka/model/status/KafkaConnectorStatus.java

```diff
@@ -39,4 +40,14 @@
     public void setConnectorStatus(Map<String, Object> connectorStatus) {
         this.connectorStatus = connectorStatus;
     }
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @Description("The maximum number of tasks for the Kafka Connector")
```

Comment:
I think we are missing a full stop here

Reply:
The `DocGenerator` will add a full stop if one is missing.